### PR TITLE
Update website cc pages.

### PR DESCRIPTION
**Pull request by EsenDR**
**Please describe the changes this PR makes and why it should be merged:**

This is an update to the website where the leading comment info has been put to the top with the trigger type, trigger, and usages all being clear where everyone can see them. Also added `parseArgs` snippet, and updated `deathmatch` cc.

**Status**

- [x] Code changes have been tested on an instance of YAGPDB, or there are no code changes
- [x] I have read and followed the [contribution guide](../CONTRIBUTING.md)

### CODE REVIEW

File: yagpdb-cc-website/docs/afk/leaveFeed.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/afk/leaveFeed.md:8]:
```suggestion
It is optional, meaning that the other CCs in this system will work fine without it - though we recommend adding it anyway, to keep your database neat and clean.
```
Would add this, can already see people asking "why would I need that"

File: yagpdb-cc-website/docs/birthday/birthday.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/birthday/birthday.md:13]:
```suggestion
Commands and usage is outlined in the [README](birthdayREADME/#commands).
```
A little nag

File: yagpdb-cc-website/docs/funccs/maze.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/funccs/maze.md:6]:
```suggestion
This command sends a maze with an optional amount crossings/bridges. It also includes a link to a downloadable solution and intuitive `execCC` support.
```

File: yagpdb-cc-website/docs/funccs/slotmachine.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/funccs/slotmachine.md:6]:
```suggestion
Slot machine game by Pedro Pessoa. Gambling can be addictive. 
```
:wink:

File: yagpdb-cc-website/docs/giveaway/basic/basicgiveaway.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/giveaway/basic/basicgiveaway.md:6]:
```suggestion
Main Giveaway CC. Supports intuitive `execCC` usage.
```

File: yagpdb-cc-website/docs/giveaway/basic_v2/basicv2giveaway.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/giveaway/basic_v2/basicv2giveaway.md:6]:
```suggestion
Main Giveaway V2 CC. Supports intuitive usage via `execCC`.
```

File: yagpdb-cc-website/docs/info/user.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/info/user.md:6]:
```suggestion
This command allows you to view information about a given user, defaulting to yourself.
```
See above

File: yagpdb-cc-website/docs/info/avatar.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/info/avatar.md:6]:
```suggestion
This command allows you to view the avatar of a given user, defaulting to yourself.
```
Dunno, feel like this eases the flow and prevents misunderstandings. Feel free to ignore.

File: yagpdb-cc-website/docs/leveling/rank.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/leveling/rank.md:5]:
```suggestion
This command manages viewing the rank of yourself, or a given member.
```

File: yagpdb-cc-website/docs/moderation/slowmode.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/moderation/slowmode.md:6]:
```suggestion
This command adds slowmode functionality: It'll delete an user's message if their cooldown isn't over yet. Configurable permission requirement to bypass the cooldown, as well as required permission to invoke the slow mode.
```

File: yagpdb-cc-website/docs/snippets/selectionsort.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/snippets/selectionsort.md:7]:
```suggestion
The array input is `$arr`. It is sorted in place.
```

File: yagpdb-cc-website/docs/snippets/parseflags.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/snippets/parseflags.md:9]:
```suggestion
    ```
    -command -m 123 positional arg -f
    ```
might be parsed into:
```
weird bullets + indentation, code block is imo neater to look at

File: yagpdb-cc-website/docs/snippets/parseflags.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/snippets/parseflags.md:15]:
```suggestion
    ```
    positional: ["positional" "arg"]  
	  flags: map {  
	    "m" => "123"  
	    "f" => true  
	  }
	  ```
```
could we get a codeblock here?

File: yagpdb-cc-website/docs/suggestion/suggestion.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/suggestion/suggestion.md:13]:
```suggestion
**Usage:** Refer to [README](suggestionREADME.md).
```

File: yagpdb-cc-website/docs/suggestion/suggestion.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/suggestion/suggestion.md:16]:
```suggestion
Refer to the [README](suggestionREADME.md) for an explanation about the configuration variables.
```

File: yagpdb-cc-website/docs/utilities/cc2file.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/utilities/cc2file.md:7]:
```suggestion
You can specify more than one ID or trigger, and the bot will generate one file per input parameter. If none are provided, output will be the entire CC list.
```
typo boye

File: yagpdb-cc-website/docs/utilities/colorpreview.md
Comment (l-zeuch) [inline yagpdb-cc-website/docs/utilities/colorpreview.md:6]:
```suggestion
This command allows you to preview how colors look. Converts hex to dec, and vice versa.
```
nitpicking :')

**Review by l-zeuch — CHANGES_REQUESTED** [review]
Well, going through 86 files and over a thousand added lines was certainly not fun. Here's me nagging on things :grin:

**Comment by EsenDR** [comment]
Yeah, sorry for the massive amounts of files. Not fun to have to go through and change/add. Thanks for going through and helping, mate!

**Review by l-zeuch — APPROVED** [review]
lgtm now, I'd say we wait for another pair of eyes

File: yagpdb-cc-website/docs/utilities/edit.md
Comment (jo3-l) [inline yagpdb-cc-website/docs/utilities/edit.md:11]: funky formatting here (I know it's probably from the original top comment) but we can fix it a bit here:
```suggestion
This command is a tool for editing messages sent by YAGPDB.

**Flags:**
`-content`: To Change Content  
`-title`, `-desc`, `-image`, `-thumbnail`, `-url`, `-author`, `-authoricon`, `-authorurl`, `-footer`, `-footericon`, `-color`: To Edit Embed  
`-force`: Makes a new embed with provided fields and discards previous embed (default behavior is simply editing provided fields of embed while preserving other fields)  
`-clrembed`: To remove the embed from a message previously containing embed (so that now it has only content. Note, You cant remove embed if content is also null)
```

**Review by jo3-l — CHANGES_REQUESTED** [review]

**Comment by EsenDR** [comment]
connect4 is added so it's best to check that as well. Also fixed a redirect that was broken.

**Review by jo3-l — APPROVED** [review]
Connect4 changes look fine to me, but gonna wait for Luca to have a chance to check 'em before merging. Thanks for the work on this!

**Review by l-zeuch — APPROVED** [review]
Thanks!

**Pull request merged**
